src/form-builder/components/FormConditionsModal.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { JSHINT } from 'jshint';
+import ObsControlScriptEditorModal from 'form-builder/components/ObsControlScriptEditorModal';
+
+window.JSHINT = JSHINT;
```

Comment:
is this required here? ObsControlScriptEditorModal has it already?

Reply:
Not required. removed here